### ISSUE TITLE
[Keyboard a11y] Edit/Cancel playlist details buttons

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/avalon#6356]
### Description

From accessibility checks performed in https://github.com/avalonmediasystem/avalon/issues/6312, it was identified that the edit and cancel playlist details buttons are not reachable with `Tab` navigation in the edit playlist page.
To observe this behavior;
1. When edit form is closed: use `Tab` navigation to reach the edit playlist details button (next to `Playlist Details` text)
2. When edit form is open: use `Tab` navigation to reach the `Cancel` button at the end of the playlist details form to close the form

In Firefox, go to `Accessibility` tab in the dev-tools and check `Show Tabbing Order` to see the tab order of the elements on the page. Observe that both edit playlist details and cancel buttons are not in the tab order

### Done Looks Like
- [ ] A keyboard user can reach and activate the edit playlist details button
- [ ] A keyboard user can reach and activate the cancel changes made to the playlist details form
- [ ] These buttons are included in the page's tab order when visible

**Comment by joncameron** [comment]
Working great! 👍
